src/System.Net.Security/tests/UnitTests/System/Security/Authentication/AuthenticationExceptionTest.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.Authentication;
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    public class AuthenticationExceptionTest
+    {
+        [Fact]
+        public void Constructor_NoParameter_DefaultMessageCorrect()
+        {
+            AuthenticationException authenticationException = new AuthenticationException();
```

Comment:
We shouldn't be testing the built in exception messages because we can't assume they are in English. In future we may want to localize .NET Core. Such tests don't add much value anyway. If you want the code coverage you can just check the Message is not empty.

Same in other file.